src/SamuraiScan/SamuraiScan.ts

```diff
@@ -42,6 +46,8 @@ export class SamuraiScan extends Madara {
 
     override hasAdvancedSearchPage = true
```

Comment:
You can remove this, since it has been changed within the generic with last commit.